<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutPymakr</class>
 <widget class="QDialog" name="AboutPymakr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>304</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>40</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pycom Pymakr</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVersion">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>80</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Version: 1.0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDescription">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>132</y>
     <width>394</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;An editor to work with Pycom products.&lt;/p&gt;&lt;p&gt;This program makes use of the following Open Source Software:&lt;br/&gt;&lt;a href=&quot;http://www.qt.io&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Qt&lt;/span&gt;&lt;/a&gt;, &lt;a href=&quot;https://www.riverbankcomputing.com/software/pyqt&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;PyQt&lt;/span&gt;&lt;/a&gt;, and &lt;a href=&quot;https://www.riverbankcomputing.com/software/qscintilla&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;QScintilla2&lt;/span&gt;&lt;/a&gt;, available under the GNU GPL v3 license.&lt;/p&gt;&lt;p&gt;Based on &lt;a href=&quot;http://eric-ide.python-projects.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Eric IDE&lt;/span&gt;&lt;/a&gt;.&lt;br/&gt;Copyright (c) 2002 - 2016 Detlev Offenbach.&lt;/p&gt;&lt;p&gt;Aditional portions Copyright (c) 2016 &lt;a href=&quot;http://www.pycom.io/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Pycom&lt;/span&gt;&lt;/a&gt;.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="imgIcon">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>32</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutPymakr</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutPymakr</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
